misc/web/novnc/include/ui.js

```diff
@@ -1260,7 +1260,7 @@ var UI;
 
         // Display the desktop name in the document title
         updateDocumentTitle: function(rfb, name) {
-            document.title = name + " - noVNC";
+            //document.title = name + " - noVNC";
```

Comment:
vnc_auto.html sets the window title to the vm name, iirc this gets overwritten when calling UI.connect() so I commented out the update